common/handy/maps.go

```diff
@@ -32,3 +32,26 @@ func (m Map[K, V]) Values() []V {
 
 	return values
 }
+
+// DefaultMap wrapper of the map that allows setting default return value on missing keys.
+type DefaultMap[K comparable, V any] struct {
+	Map[K, V]
```

Comment:
Cool! I'd name this data to be a little more explicit about what is being held here.

Reply:
This uses embedding. Which means all methods that `Map` has will be available for `DefaultMap`. This makes it a true wrapper. A Map with a perk. This means we can use Keys(), Values() and no need to reimplement them. I think they compose nicely.
I will add a comment about this.